scripts/build_review_app.sh

```diff
@@ -73,9 +73,16 @@ hokusai review_app env copy $NAME --configmap nginx-config --verbose
 # authentication requests won't work!
 hokusai review_app env set $NAME \
   APP_URL="https://$NAME.artsy.net" \
+  APPLICATION_NAME="$NAME" \
   COOKIE_DOMAIN="$NAME.artsy.net" \
   FORCE_URL="https://$NAME.artsy.net"
 
+# Publish Force assets to S3
+hokusai review_app run $NAME 'yarn publish-assets'
```

Comment:
Did you consider/try out a `hokusai review_app deploy ...`? I think that would be more resilient to change as it would run any deploy lifecycle events, regardless of what they are at any given time.